src/config/tokens/core/_core-opacity.scss

```diff
@@ -7,6 +7,7 @@ $opacity-0: 0;
 $opacity-10: 0.1;
 $opacity-20: 0.2;
 $opacity-30: 0.3;
+$opacity-38: 0.38;
```

Comment:
Is the difference between 0.38 and 0.40 really noticeable?

Reply:
i guess we can debate on it :)